recipes/trio-parallel/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "trio-parallel" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 32fad6a7c9ba480f70d424772506fe4299c5dcc42c7628e6fd49df9cf6bd32e4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python =>3.6
+    - pip
+  run:
+    - python =>3.6
+    - trio >=0.18.0
+    - outcome
+
+test:
+  imports:
+    - trio_parallel
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://trio-parallel.readthedocs.io/
+  license: MIT OR Apache-2.0
+  license_family: OTHER
+  license_file: LICENSE
```

Let's include all 3 files:
```suggestion
  license_file:
    - LICENSE
    - LICENSE.APACHE2
    - LICENSE.MIT
```